crates/compiler/load_internal/src/file.rs

```diff
@@ -3231,6 +3246,24 @@ fn finish_specialization<'a>(
     })
 }
 
+fn proc_layout_for<'a>(
+    mut proc_symbols: impl Iterator<Item = (Symbol, ProcLayout<'a>)>,
+    symbol: Symbol,
+) -> ProcLayout<'a> {
+    match proc_symbols.find(|(s, _)| *s == symbol) {
+        Some((_, layout)) => layout,
```

Comment:
I know this was here before, but in the future, we might want to check that there is only one specialization for the symbol here - if there are multiple, the developer either has multiple host-exposed symbols at least one of which has >1 specialization, or it's a compiler bug

Reply:
Good call!